src/hooks/useAccountTransactions.js

```diff
@@ -45,23 +52,23 @@ export default function useAccountTransactions(initialized, isFocused) {
   const { sections } = buildTransactionsSectionsSelector(accountState);
 
   const remainingItemsLabel = useMemo(() => {
-    const remainingLength = transactions.length - slicedTransaction.length;
+    const remainingLength = allTransactions.length - slicedTransaction.length;
     if (remainingLength === 0) {
       return null;
     }
-    if (transactions.length - slicedTransaction.length <= NOE_PAGE) {
+    if (allTransactions.length - slicedTransaction.length <= NOE_PAGE) {
```

Comment:
Perhaps it is okay to use `remainingLength` there making it `remainingLength <= NOE_PAGE`